build-farm/platform-specific-configurations/linux.sh

```diff
@@ -74,7 +74,7 @@ fi
 
 if [ "${ARCHITECTURE}" == "arm" ]
 then
-  export CONFIGURE_ARGS_FOR_ANY_PLATFORM="--with-jobs=4 --with-memory-size=2000"
+  export CONFIGURE_ARGS_FOR_ANY_PLATFORM="$CONFIGURE_ARGS_FOR_ANY_PLATFORM --with-jobs=4 --with-memory-size=2000"
```

Comment:
probably should have braces {}

Reply:
Done, and I changed the others in the "arm" section as well, to makie it consistent.